src/web/tntnet.xml

```diff
@@ -356,7 +356,7 @@
     <mapping>
       <target>input_power_chain@bios_web</target>
       <method>GET</method>
-      <url>^/api/v1/topology/input_power_chain/([0-9]+)$</url>
+      <url>^/api/v1/topology/input_power_chain/.*$</url>
```

Comment:
WRONG FIX

1) Must be in parentheses (`$1` used below)
2) If we need this not-empty, `.+` at least
3) Do we really want ANY chars? Or same as in CI `[-.a-z0-9]`?